[2024-07-17T07:48:09.650+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T07:48:07.894712+00:00 [queued]>
[2024-07-17T07:48:09.659+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T07:48:07.894712+00:00 [queued]>
[2024-07-17T07:48:09.659+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-07-17T07:48:09.672+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): Pre-Scrape_script> on 2024-07-17 07:48:07.894712+00:00
[2024-07-17T07:48:09.677+0000] {standard_task_runner.py:60} INFO - Started process 110 to run task
[2024-07-17T07:48:09.678+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Webscraping', 'Pre-Scrape_script', 'manual__2024-07-17T07:48:07.894712+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/webscrape_dag.py', '--cfg-path', '/tmp/tmpixtesz46']
[2024-07-17T07:48:09.679+0000] {standard_task_runner.py:88} INFO - Job 128: Subtask Pre-Scrape_script
[2024-07-17T07:48:09.734+0000] {task_command.py:423} INFO - Running <TaskInstance: Webscraping.Pre-Scrape_script manual__2024-07-17T07:48:07.894712+00:00 [running]> on host 11bf965559b0
[2024-07-17T07:48:09.818+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Webscraping' AIRFLOW_CTX_TASK_ID='Pre-Scrape_script' AIRFLOW_CTX_EXECUTION_DATE='2024-07-17T07:48:07.894712+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-17T07:48:07.894712+00:00'
[2024-07-17T07:48:09.821+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-17T07:48:09.821+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /webscrape_scripts/pre_scrape.py']
[2024-07-17T07:48:09.830+0000] {subprocess.py:86} INFO - Output:
[2024-07-17T07:48:09.852+0000] {subprocess.py:93} INFO - python3: can't open file '/webscrape_scripts/pre_scrape.py': [Errno 2] No such file or directory
[2024-07-17T07:48:09.854+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-07-17T07:48:09.867+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-07-17T07:48:09.869+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=Webscraping, task_id=Pre-Scrape_script, execution_date=20240717T074807, start_date=20240717T074809, end_date=20240717T074809
[2024-07-17T07:48:09.882+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 128 for task Pre-Scrape_script (Bash command failed. The command returned a non-zero exit code 2.; 110)
[2024-07-17T07:48:09.893+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-07-17T07:48:09.917+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
